acsg.js

```diff
@@ -360,9 +360,9 @@ acsg.World = (function () {
 
   World.prototype.serialize = function () {
     var players = []
-        ,food = []
-        ,states = []
-        ,i
+      ,food = []
+      ,states = []
+      ,i
 
```

Comment:
[Descending to the truly trivial...] I saw this as a suggestion, and I like it better (and didn't find it cumbersome to type):
```
    var players = []
      , food = []
      , states = []
      , i
```

Reply:
That looks quite reasonable to me.